src/OTALKRES.cpp

```diff
@@ -1087,7 +1087,7 @@ int TalkRes::detect_talk()
 		choice_question_second_line = NULL;
 
 		talk_choice_count = 0;
-		add_talk_choice( _("Continue"), 0 );
+		add_talk_choice( _("Continue."), 0 );
```

Comment:
What rule does this follow? This is more of a "button" than a sentence.
Yes I know that this change is consistent with the rest of the talk choices.